shared/Models/SensorStatus.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+enum SensorStatus: String, Codable {
```

Comment:
Please add a comment to all cases. You can use the API's documentation (https://skopje.pulse.eco/restapi) for the old statuses and something like 'Used as a default value' for the unknown case.